catalog/view/stylesheet/securesubmit.css

```diff
@@ -1,4 +1,15 @@
-/*Makenzie*/
+label {
+  text-transform:uppercase;
+}
+.form-group:nth-child(2),
+.form-group:nth-child(3) {
+  width:49%;
+  display:inline-block;
+}
+.form-group:nth-child(3) {
+  float:right;
+}
+
```

Comment:
These statements should be be altered so that they are specific to Heartland's elements. As it stands, they could affect other elements on the page.

Reply:
css updated